### DISCUSSION TITLE
WASP does not seem to be available.

### DISCUSSION THREAD

**Discussion by libinbabu1991** [idaholab/moose#28931] [Q&A Getting Started] [answered]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

Hi, 
I was trying to install Moose on a new system. I installed Ubuntu app and miniconda following the instructions on the Moose website. 

However, when I try to compile the make file, it gives error about the WASP as below.

(moose) root@123-SPARE-LAPTOP:~/projects/moose# cd ~/projects/moose/test
(moose) root@123-SPARE-LAPTOP:~/projects/moose/test# make -j 6

***ERROR***
WASP does not seem to be available.

I tried to install WASP following instructions from a forum. It shows another issue about not being in the conda environment.
conda install moose-wasp

Please advise how to fix this errors.

Thanks.

**Comment by milljm** [comment]
There are a few phrases you are using that is throwing me off...

Is this a Docker image (asking because it seems you logged in as root).
Also, was this a copy/paste mishap, or does the following really exist:

```pre
(moose) root@123-SPARE-LAPTOP:~/projects/moose# cd /projects/moose/test
(moose) root@123-SPARE-LAPTOP:/projects/moose/test# make -j 6
```
I need to scrutinizing each line:
```pre
(moose) root@123-SPARE-LAPTOP:~/projects/moose# cd /projects/moose/test
                              ^                    ^ you're leaving $HOME/projects
                              ^ indicates you're in $HOME/projects


(moose) root@123-SPARE-LAPTOP:/projects/moose/test# make -j 6
                              ^ and it exists, meaning you have two locations of moose maybe?
```
Is the above correct? You have two locations of MOOSE?

- /projects/moose
- ~/projects/moose

That might indicate an issue. Perhaps not. But I would like to know which location we're truly dealing with before we dig too deep. If this is a Docker image, then it's possible you're running into an issue with Conda not being fully available.

New system, installed Ubuntu app? Is this a WSL instance then? As in, the windows machine is brand new, and you installed the Ubuntu app? If this is the case, I would recommend avoiding running as fully fledged `root` user. You're welcome to do to so, but it is highly discouraged if possible.

Can you please describe what sort of system we're dealing with; WSL or Docker, or full Ubuntu native Linux, and the interesting issue of possibly having two versions of MOOSE, and then we can begin

**Reply by libinbabu1991** [reply]
Hi, 
This is an Ubuntu app with WSL from Micosoft store.
I believe I have moose installed only in the project folder which is located in the root.

**Reply by milljm** [reply]
If you wish to run as root, thats on you. There should be no need, but I'll not question why you are doing it.

I think you have two locations of MOOSE, since the prompt allowed you to be in both locations without error. Assuming you want to use `/projects/moose`, we will stick to that. Just pointing out that we need to be carful:
```bash
cd ~/projects/moose
# should land you in /root/projects/moose (so long as you are logged in as root)
# while:
cd /projects/moose
# is literally /projects/moose for all users.
```
`~` is shorthand for `$HOME` (a users home directory).

Anyways...
Can you provide the following results:
```bash
cd /projects/moose/scripts
./diagnostics.sh
```
I wish to see the current running environment...

**Comment by libinbabu1991** [comment, marked as answer]
Hi, 
You were right, I had installed Moose in the root/projects folder and in C/Fore_moose/projects/moose folder.

I installed conda in the second location and it is working now. I was able to run one example successfully. 
 Thanks for your assistance.
